biztown/src/components/AboutSection.js

```diff
@@ -1,241 +1,298 @@
-                            type="text"
-                            defaultValue={this.props.data.teacher}
-                            onChange={this.props.handleChange}
-                        />
-                    </div>
-                </li>
-                <li className="form-group">
-                    <div className="flex-column-container">
-                        <label>NOT including this session, how many times have you participated in JA in the past?</label>
-                        <div className="multiple-choice-answer-container">
-                            <ParticipationComponent
-                                participation={this.props.data.participation}
-                                handleChange={this.props.handleChange}
-                            />
-                        </div>
-                    </div>
-                </li>
-            </ol>
-            </div>
-        );
+  render() {
+    if (!Object.keys(this.props.schoolData).length) {
+      DataManager.getAll("schools").then((data) => {
+        var schoolMap = {};
+        data.results.forEach((school) => {
+          if (!schoolMap[school.district]) {
+            schoolMap[school.district] = [];
+          }
+          schoolMap[school.district].push(school);
+        });
+        this.props.setSchoolData(schoolMap);
+      });
     }
-}
 
-function GradeComponent(props) {
-    var gradeInput = gradeOptions.map(gradeOption => {
-        return (
-            <div
-                className="flex-container"
-                key={gradeOption}
-            >
+    return (
+      <div>
+        <h3>Tell Us About You</h3>
+        <ol>
+          <li className="form-group">
+            <div className="flex-column-container">
+              <label className="form-question">
+                What are the first three letters of your last name?
+              </label>
+              <input
+                className="form-control form-control-text-input"
+                id="name"
+                name="name"
+                type="text"
+                defaultValue={this.props.data.name}
+                maxLength="3"
+                onChange={this.props.handleChange}
+                required
+              />
+            </div>
+          </li>
+          <li className="form-group">
+            <div className="flex-column-container">
+              <label className="form-question">
+                When were you born?
+              </label>
+              <div>
                 <input
-                    className="form-control-radio-btn"
-                    type="radio"
-                    id="grade"
-                    name="grade"
-                    value={gradeOption}
-                    checked={props.grade === gradeOption}
-                    onChange={props.handleChange}
+                  className="form-control"
+                  id="birthDate"
+                  name="birthDate"
+                  type="date"
```

Comment:
It is worth considering doing the TODO item and updating this to use react-dates or something, because at the minute after cross-browser testing, it appears as a free text input field in safari.

Reply:
Added to the backlog.